pgt/src/PGT/Output/Test/Result/examples/json-rows.expected

```diff
@@ -6,4 +6,4 @@
  pri_metrics  | available_bar                                        | {                                                                     +| pri_metrics.bar
               |                                                      |     "public.timestamp_not_null"                                       +|
               |                                                      | }                                                                      |
-(2 row)
+(2 rows)
```

Comment:
Comment: Please take note that the row count in `pgt/src/PGT/Output/Test/Result/examples/json-rows.expected.parsed` is generated from the returned row lines. A future change makes it impossible to have the kind of mismatch is `row/rows` spelling on  DB results.